distsql/xeval/eval.go

```diff
@@ -35,6 +35,12 @@ const (
 	compareResultNull = -2
 )
 
+// Flags
+const (
+	FlagIgnoreTruncate    uint64 = 1
+	FlagTruncateAsWarning uint64 = 1 << 1
+)
```

Comment:
This flag is used by both TiDB and TiKV, we better make it explicit and consistent.